### ISSUE TITLE
RegularPolygonMarker needs a docstring

### ISSUE THREAD

**Issue by ocefpaf** [python-visualization/folium#281]
Things like  `opacity` and `fill_opacity`, `color` and `fill_color` requires better explanation.

```python
class RegularPolygonMarker(MacroElement):
    def __init__(self, location, popup=None,
                 color='black', opacity=1, weight=2,
                 fill_color='blue', fill_opacity=1,
                 number_of_sides=4, rotation=0, radius=15):
        """
        TODO docstring here

        """
```

Reference: https://github.com/humangeo/leaflet-dvf/wiki/6.-Markers

@jelmelk is this something you are willing to do?

**Comment by ocefpaf** [comment]
Also, the mapping from the old to the new syntax might help to sort this out.

```python
        marker = RegularPolygonMarker(location,
                                      popup=popup_,
                                      color=line_color,
                                      opacity=line_opacity,
                                      weight=line_weight,
                                      fill_color=fill_color,
                                      fill_opacity=fill_opacity,
                                      number_of_sides=num_sides,
                                      rotation=rotation,
                                      radius=radius)
```

**Comment by BibMartin** [comment]
Fixed by #282. Closing this.
